appservice/src/intTest/groovy/nu/yona/server/BuddyValidationTest.groovy

```diff
@@ -166,7 +166,7 @@ class BuddyValidationTest extends AbstractAppServiceIntegrationTest
 		User richard = addRichard()
 		User bob = addBob()
 		bob.emailAddress = "bob@dunn.com"
-		appService.sendBuddyConnectRequest(richard, bob)
+		assertResponseStatusCreated(appService.sendBuddyConnectRequest(richard, bob))
```

Comment:
Isn't it more logical to place this assertion inside the `sendBuddyConnectRequest` method?

Reply:
:anguished: It's already part of it. The last parameter is a boolean named ``assertSuccess``, with ``true`` as default. I'll rever this change.